src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeArm.java

```diff
@@ -0,0 +1,42 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import java.util.Collections;
+import java.util.LinkedList;
+
+public class NodeArm {
+    private NodeArm parent;
+    private int id;
+    private final LinkedList<NodeArm> neighbors;
+    private final double anglePos;//problem
+    private final double extendPos;//problem
+    public NodeArm(int id, double x, double y){
```

Comment:
Raz - what is x, y?

Reply:
changed names to anglePos and extenderPos